incubator/redash/values.yaml

```diff
@@ -0,0 +1,173 @@
+    #  memory: 128Mi
+    requests:
+    #  cpu: 100m
+      memory: 2Gi
+
+  ## Node labels for server pod assignment
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## Tolerations for server pod assignment
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+
+  ## Affinity for server pod assignment
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
+  affinity: {}
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts:
+    - chart-example.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+## Configuration for Redash ad-hoc workers
+##
+adhocWorker:
+  ## Ad-hoc worker specific environment variables
+  ## Add any environment variables from https://redash.io/help-onpremise/setup/settings-environment-variables.html
+  ##
+  env:
+    QUEUES: "queries,celery"
+    WORKERS_COUNT: 2
+
+  replicaCount: 1
+
+  ## Ad-hoc worker resource requests and limits
+  ## Ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources: {}
+
+  ## Node labels for ad-hoc worker pod assignment
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## Tolerations for ad-hoc worker pod assignment
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+
+  ## Affinity for ad-hoc worker pod assignment
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
+  affinity: {}
+
+## Configuration for Redash scheduled workers
+##
+scheduledWorker:
+  ## Scheduled worker specific environment variables
+  ## Add any environment variables from https://redash.io/help-onpremise/setup/settings-environment-variables.html
+  ##
+  env:
+    QUEUES: "scheduled_queries"
+    WORKERS_COUNT: 2
+
+  replicaCount: 1
+
+  ## Scheduled worker resource requests and limits
+  ## Ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources: {}
+
+  ## Node labels for scheduled worker pod assignment
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## Tolerations for scheduled worker pod assignment
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+
+  ## Affinity for scheduled worker pod assignment
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
+  affinity: {}
+
+## Configuration values for the postgresql dependency
+## This PostgreSQL instance is used for all Redash state storage
+## ref: https://github.com/kubernetes/charts/blob/master/stable/postgresql/README.md
+##
+postgresql:
```

Comment:
Is there a way to define an external postgresql database instead of running its own? for setups that already have a DB running (i.e. using AWS managed service) this would remove the need for persistence too.